### DISCUSSION TITLE
.exr and .hdr files have been looking dark

### DISCUSSION THREAD

**Discussion by VitalSkib** [d2phap/ImageGlass#2044] [General]
I can't figure out what's wrong with my setup, but my .exr and .hdr files have been looking dark for a while now. 
What can I do to fix this? 
Thanks.

Here's an example of the same .exr file opened in the IG (looks dark) and in the [QuickLook](https://github.com/QL-Win/QuickLook) (looks correct).

![IG-](https://github.com/user-attachments/assets/8ae36754-130f-4c29-82d0-f7c986721703)
![QL](https://github.com/user-attachments/assets/3a639405-0f29-4f6e-bdb1-6772dd3641c8)

**Comment by d2phap** [comment]
@VitalSkib can you attach the sample image?

You may want to try changing these settings as well
![image](https://github.com/user-attachments/assets/9352dd5d-e4d5-44d7-8ca9-122ec075746d)

**Reply by VitalSkib** [reply]
> @VitalSkib can you attach the sample image?

Of course, here is this file:

[panorama.zip](https://github.com/user-attachments/files/17965032/panorama.zip)


> You may want to try changing these settings as well

No, unfortunately it doesn't have any effect. 
I even tried deleting the config file, and even completely reinstalled IG..
